spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1726,6 +1726,18 @@ A client for the Spring Cloud Stream schema registry can be configured using the
   }
 ----
 
+The default configuration will create a `DefaultSchemaRegistryClient` bean. If you want to use the confluent schema registry, you need to create a bean of that type
```

Comment:
We should provide a reference to the Confluent schema registry documentation (ideally for a version that we know we support). 
Sentence needs to be finished. 
Also we can also be more specific as to what 'that type' is (i.e. `ConfluentSchemaRegistryClient`).